packages/@aws-cdk/integ-runner/lib/runner/runner-base.ts

```diff
@@ -64,6 +73,41 @@ export interface IntegRunnerOptions {
  * Represents an Integration test runner
  */
 export abstract class IntegRunner {
+
+  // Best-effort reconstruction of the classpath of the file in the project.
+  // Example: /absolute/file/path/src/main/java/com/myorg/IntegTest.java -> com.myorg.IntegTest
```

Comment:
The `src/test` is usually the location of unit tests in Java, and doing it this way would allow users to separate them under different directories (for example `src/integ` or `src/integtest`). I think it's best not to make too many assumptions about the directory structure (other than the package root being under `java/` and some `pom.xml` file exists somewhere in the hierarchy)

Reply:
Yeah I didn't make any assumptions about how the project is structured. From what we know, the path to the files can be `src/test`, `src/tests`, `src/integ`, `src/tests/integ` or anything else.